test_scripts/RC/OnRCStatus/commonOnRCStatus.lua

```diff
@@ -254,6 +264,12 @@ function m.validateOnRCStatusForApp(pAppId, pExpData)
       m.sortModules(d.payload.allocatedModules)
       return compareValues(pExpData, d.payload, "payload")
     end)
+  :ValidIf(function(_, d)
+    if d.payload.allowed and false == pAllowed then
```

Comment:
@GetmanetsIrina In case if `d.payload.allowed` = `true` and `pAllowed` = `false` test case will be failed.
Presence of some argument should be verified with ` d.payload.allowed` ~= `nil` code